clients/client-python/setup.py

```diff
@@ -15,7 +15,7 @@
 setup(
     name="gravitino",
     description="Python lib/client for Gravitino",
-    version="0.5.0.dev21",
+    version="0.5.2.dev1",
```

Comment:
CC @FANNG1 this should be updated to "0.5.1" before creating a tag.

Reply:
ok, after creating a tag, the python version should be reverted to `0.5.2.dev1`?

Reply:
Yes.